test/unit/core/type_traits/range_iterator_test.cpp

```diff
@@ -120,17 +120,15 @@ TEST(range_and_iterator, rvalue_reference_)
     using iterator_of_int_vector = std::ranges::iterator_t<std::vector<int>>;
     using foreign_iterator = seqan3::detail::random_access_iterator<std::vector<int>>;
     auto v = std::views::iota(1);
-    using type_list_example = seqan3::type_list<seqan3::rvalue_reference_t<std::vector<int>>, // short
-                                                typename seqan3::rvalue_reference<std::vector<int>>::type, // long
```

Comment:
Why did you delete these two lines?

Reply:
These lines explicitly test the behaviour of the seqan3::rvalue_reference struct that is being replaced. But maybe the complete delete isn't the way to go, @marehr ?

Reply:
For this expression

```c++
typename seqan3::rvalue_reference<std::vector<int>>::type, // long
```

there is no equivalent in the standard, because they only support the short notation `seqan3::rvalue_reference_t`.

See https://en.cppreference.com/w/cpp/ranges/iterator_t

So we just remove this test case.

Reply:
Ok, thanks for the clearification. :)